frontstage/views/surveys/help/surveys_help.py

```diff
@@ -9,10 +9,54 @@
 
 from frontstage.common.authorisation import jwt_authorization
 from frontstage.controllers import survey_controller, conversation_controller
-from frontstage.models import HelpOptionsForm, HelpCompletingThisSurveyForm, SecureMessagingForm
+from frontstage.models import HelpOptionsForm, HelpCompletingThisSurveyForm, SecureMessagingForm, \
+    HelpInfoAboutThisSurveyForm
 from frontstage.views.surveys import surveys_bp
 
 logger = wrap_logger(logging.getLogger(__name__))
+help_completing_this_survey_title = "Help completing this survey"
+info_about_this_survey_title = "Information about this survey"
+option_template_url = {
```

Comment:
Super minor, but `option_template_url` makes me think the variable is a string.  Something like `option_template_urls` or `option_template_url_mapping` (or something along those lines) would work better I think.

Reply:
makes sense.. will change it